admin-dev/themes/default/js/bundle/admin_parameters/performancePageUI.js

```diff
@@ -46,7 +46,7 @@ var PerformancePageUI = {
 
         if(cacheEnabledInput.value === CACHE_ENABLED) {
             for (var i = 0; i < cachingElements.length; i++) {
-                cachingElements[i].style.display = "block";
+                cachingElements[i].style.display = "";
```

Comment:
Is this block still needed?

Reply:
Yep, it's used for toggling related options visible/invisible depending of the chosen option. Ofc this script should be moved to the new theme, but that's a subject for another PR.